storage/innobase/xtrabackup/utils/percona-xtrabackup-8.0_builder.sh

```diff
@@ -243,9 +243,9 @@ install_deps() {
             PKGLIST+=" binutils-devel python3-pip python3-setuptools"
             PKGLIST+=" libcurl-devel cmake libaio-devel zlib-devel libev-devel bison make gcc"
             PKGLIST+=" rpm-build libgcrypt-devel ncurses-devel readline-devel openssl-devel gcc-c++"
-            PKGLIST+=" vim-common rpmlint patchelf python3-sphinx python3-wheel"
+            PKGLIST+=" vim-common rpmlint patchelf python3-wheel"
```

Comment:
may I know why this is changed?
We require sphynix only on RHEL 9/OEL9?

Reply:
Hi @satya-bodapati 
This change is OK for the build script.

Reply:
Ok